sdk/keyvault/Azure.Security.KeyVault.Keys/api/Azure.Security.KeyVault.Keys.netstandard2.0.cs

```diff
@@ -184,6 +209,7 @@ public static partial class KeyModelFactory
         public static Azure.Security.KeyVault.Keys.KeyProperties KeyProperties(System.Uri id, System.Uri vaultUri, string name, string version, bool managed, System.DateTimeOffset? createdOn, System.DateTimeOffset? updatedOn, string recoveryLevel) { throw null; }
         public static Azure.Security.KeyVault.Keys.KeyProperties KeyProperties(System.Uri id = null, System.Uri vaultUri = null, string name = null, string version = null, bool managed = false, System.DateTimeOffset? createdOn = default(System.DateTimeOffset?), System.DateTimeOffset? updatedOn = default(System.DateTimeOffset?), string recoveryLevel = null, int? recoverableDays = default(int?)) { throw null; }
         public static Azure.Security.KeyVault.Keys.KeyVaultKey KeyVaultKey(Azure.Security.KeyVault.Keys.KeyProperties properties, Azure.Security.KeyVault.Keys.JsonWebKey key) { throw null; }
+        public static Azure.Security.KeyVault.Keys.ReleaseKeyResult ReleaseKeyResult(string value, Azure.Security.KeyVault.Keys.KeyExportEncryptionAlgorithm? algorithm) { throw null; }
```

Comment:
FYI in JS I removed `algorithm` from the result.

- If the user did pass one in, both the result (as written) and the decoded JWS will contain the algorithm, making it superfluous
- If the user did not pass one in, one will still be defaulted (I think CKM_RSA_AES_KEY_WRAP) but now the ReleaseKeyResult will not be truthful. I didn't realize this when adding it, but it's probably best to have the JWS be the source of truth for the result